packages/discovery-react-components/src/components/CIDocument/components/NavigationToolbar/NavigationToolbar.tsx

```diff
@@ -2,8 +2,8 @@ import React, { FC } from 'react';
 import cx from 'classnames';
 import { settings } from 'carbon-components';
 import { Button } from 'carbon-components-react';
-import ChevronLeft16 from '@carbon/icons-react/lib/chevron--left/16';
-import ChevronRight16 from '@carbon/icons-react/lib/chevron--right/16';
+import ChevronLeft16 from '@carbon/icons-react/es/chevron--left/16';
+import ChevronRight16 from '@carbon/icons-react/es/chevron--right/16';
```

Comment:
If we're making these changes anyway, I would prefer to go with the cleaner looking code:
```suggestion
import { ChevronRight16 } from '@carbon/icons-react';
```

If you don't have time to update this, I can handle it tomorrow.

Reply:
I tried using that cleaner format but for some reason I couldn't get it to build that way. It only worked for me when using the full /es/ path. If you want to take a look and see if you can figure out why you probably understand it better than I.